net/core/filter.c

```diff
@@ -3911,7 +3911,7 @@ static int bpf_xdp_mb_adjust_tail(struct xdp_buff *xdp, int offset)
 		}
 
 		if (unlikely(frags_to_free))
-			xdp_return_frags_from_buff(xdp, frags_to_free);
+			xdp_return_num_frags_from_buff(xdp, frags_to_free);
```

Comment:
@chaudron this should go to patch 1/3

Reply:
Force pushed it, but it's not picking it up. Will open a new pr.